framework/src/modules/chain/logic/bft.js

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
probably better to create
`chain/bft/headers_queue.js`

Reply:
I don't like putting everything on top level, that does not provide any advantage, rather create ambiguity in current logical structure.

Reply:
everything will be in the `chain` level so it will be solved

Reply:
Yes its should not be on chain level, as we have segerations of business logic, helpers and submodules. So we should put it where it belongs.

Reply:
i meant we won't have separation of logic and submodule anymore

Reply:
When we remove submodules and logic in the chain module, then we can refactor restructure this as well. For now its good to use the structure which is in place.